packages/features/schedules/components/Schedule.tsx

```diff
@@ -383,21 +383,39 @@ const CopyTimes = ({
       <div className="p-2">
         <p className="h6 text-emphasis pb-3 pl-1 text-xs font-medium uppercase">{t("copy_times_to")}</p>
         <ol className="space-y-2">
+        <li key="select all">
+                <label className="text-default flex w-full items-center justify-between">
+                  <span className="px-1">{t('select_all')}</span>
+                  <input
+                    value={t('select_all')}
+                    checked={selected.length === 7}
+                    onChange={(e) => {
+                      if (e.target.checked) {
+                        setSelected([0,1,2,3,4,5,6]);
+                      } else if (!e.target.checked) {
+                        setSelected([]);
+                      }
+                    }}
+                    type="checkbox"
+                    className="border-default bg-default text-emphasis disabled:text-muted focus:ring-emphasis dark:checked:bg-muted focus:bg-default dark:checked:focus:bg-default dark:checked:hover:bg-subtle dark:checked:hover:text-inverted inline-block rounded-[4px]"
+                  />
+                </label>
+            </li>
           {weekdayNames(i18n.language, weekStart).map((weekday, num) => {
             const weekdayIndex = (num + weekStart) % 7;
             return (
               <li key={weekday}>
                 <label className="text-default flex w-full items-center justify-between">
                   <span className="px-1">{weekday}</span>
                   <input
-                    value={weekdayIndex}
-                    defaultChecked={disabled === weekdayIndex}
+                    value={weekdayIndex}                  
+                    checked={selected.includes(weekdayIndex) || disabled === weekdayIndex}
                     disabled={disabled === weekdayIndex}
                     onChange={(e) => {
                       if (e.target.checked && !selected.includes(weekdayIndex)) {
                         setSelected(selected.concat([weekdayIndex]));
                       } else if (!e.target.checked && selected.includes(weekdayIndex)) {
-                        setSelected(selected.slice(selected.indexOf(weekdayIndex), 1));
+                        setSelected(selected.filter(item => item !== weekdayIndex));
                       }
                     }}
```

Comment:
same here

Reply:
Thank you for the feedback, I just changed it to calcom component, the thing is that i kept them based on the previous development.  I did not change the label since it had a different style, I tried to keep the change on the previous code to it's minimal, or should I work on that?